template/lambda-sqs-worker-cdk/infra/appStack.ts

```diff
@@ -51,32 +54,129 @@ export class AppStack extends Stack {
       encryptionMasterKey: kmsKey,
     });
 
+    topic.addSubscription(new aws_sns_subscriptions.SqsSubscription(queue));
+
     const architecture = '<%- lambdaCdkArchitecture %>';
 
-    const worker = new aws_lambda_nodejs.NodejsFunction(this, 'worker', {
+    const defaultWorkerConfig: aws_lambda_nodejs.NodejsFunctionProps = {
       architecture: aws_lambda.Architecture[architecture],
-      entry: './src/app.ts',
-      bundling: {
-        sourceMap: true,
-        target: 'node20',
-        // By default the aws-sdk-v3 is set as an external module, however, we want it to be bundled with the lambda
-        externalModules: [],
-      },
       runtime: aws_lambda.Runtime.NODEJS_20_X,
-      functionName: '<%- serviceName %>',
       environmentEncryption: kmsKey,
+      // aws-sdk-v3 sets this to true by default so it is not necessary to set the environment variable
+      awsSdkConnectionReuse: false,
+    };
+
+    const defaultWorkerBundlingConfig: aws_lambda_nodejs.BundlingOptions = {
+      sourceMap: true,
+      target: 'node20',
+      // By default the aws-sdk-v3 is set as an external module, however, we want it to be bundled with the lambda
+      externalModules: [],
+    };
+
+    const defaultWorkerEnvironment: Record<string, string> = {
+      NODE_ENV: 'production',
+      // https://nodejs.org/api/cli.html#cli_node_options_options
+      NODE_OPTIONS: '--enable-source-maps',
+    };
+
+    const worker = new aws_lambda_nodejs.NodejsFunction(this, 'worker', {
+      ...defaultWorkerConfig,
+      entry: './src/app.ts',
+      timeout: Duration.seconds(30),
+      bundling: defaultWorkerBundlingConfig,
+      functionName: '<%- serviceName %>',
       environment: {
-        NODE_ENV: 'production',
-        // https://nodejs.org/api/cli.html#cli_node_options_options
-        NODE_OPTIONS: '--enable-source-maps',
+        ...defaultWorkerEnvironment,
         ...context.workerLambda.environment,
       },
-      // aws-sdk-v3 sets this to true by default so it is not necessary to set the environment variable
-      awsSdkConnectionReuse: false,
+      // https://github.com/aws/aws-cdk/issues/28237
+      // This forces the lambda to be updated on every deployment
+      // If you do not wish to use hotswap, you can remove the new Date().toISOString() from the description
+      description: `Lambda description - updated at ${new Date().toISOString()}`,
```

Comment:
Would we rather:

1. Encourage proper descriptions, in which case I'd advocate for a `TODO: ` in here
2. Not bother with descriptions, in which case it can just say `Updated at {now}`